app2/ota_client_service.py

```diff
@@ -17,7 +18,7 @@ def __init__(self, ota_client_stub):
 
     def Update(self, request, context):
         # TODO: conver grpc request to dict
-        results = self._stub.update(request)
+        results = asyncio.run(self._stub.update(request))
```

Comment:
@Bodong-Yang For ota caller(e.g. Iot Agent, grpc client), the Update call needs to be returned immediately, to observe the progress.